piker/clearing/_paper_engine.py

```diff
@@ -533,6 +529,11 @@ async def trades_dialogue(
         ) as feed,
 
     ):
+
+        with open_pps(broker, 'paper', False) as table:
```

Comment:
not a  big deal, but generally for extra kwarg-style opts it's good to explicitly denote them like, `write_config=False`.